<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PhoneBook</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QLabel" name="searchLabel">
        <property name="text">
         <string>Search:</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-weight: bold; color: #2d3748;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchLineEdit">
        <property name="placeholderText">
         <string>Search by name, phone or email...</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #cbd5e0; border-radius: 6px; padding: 6px 12px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="themeButton">
        <property name="text">
         <string>Dark Theme</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #4a5568; color: white; border: none; border-radius: 6px; padding: 6px 12px; font-weight: 600;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Refresh</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #6b46c1; color: white; border: none; border-radius: 6px; padding: 6px 12px; font-weight: 600;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="contactsTableView">
      <property name="styleSheet">
       <string notr="true">border: 1px solid #e2e8f0; border-radius: 6px;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add Contact</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #3182ce; color: white; border: none; border-radius: 6px; padding: 6px 12px; font-weight: 600;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="text">
         <string>Edit Contact</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #38a169; color: white; border: none; border-radius: 6px; padding: 6px 12px; font-weight: 600;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Delete Contact</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #e53e3e; color: white; border: none; border-radius: 6px; padding: 6px 12px; font-weight: 600;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Total contacts: 0</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #edf2f7; color: #4a5568; border-radius: 6px; padding: 6px 12px; font-weight: 600;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
